src/js/node/http.ts

```diff
@@ -2,13 +2,16 @@
 const EventEmitter = require("node:events");
 const { isTypedArray } = require("node:util/types");
 const { Duplex, Readable, Writable, ERR_STREAM_WRITE_AFTER_END, ERR_STREAM_ALREADY_FINISHED } = require("node:stream");
+const cluster = require("node:cluster");
```

Comment:
This adds overhead to the start time. When do we need to do this? Let's call require as late as possible. if it's only for the below usage in `reusePort`, then we should delete it entirely